content/authors/samson-njoroge/index.md

```diff
@@ -0,0 +1,8 @@
+
+title: Samson Njoroge

```

Comment:
This section isn't arranged in the proper Markdown format. Check out this [example](https://github.com/section-engineering-education/engineering-education/blob/master/content/authors/ahmad-mardeni/index.md) to fix it.